account_netting/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Vicent Cubells <vicent.cubells@serviciosbaeza.com>
```

Comment:
This email address is incorrect, but any way, we lastly prefer to remove our addresses from here. And include yourself here.

Reply:
@alexis-via can you perform this change and I merge?